include/nonunity/actions/NEML2Action.h

```diff
@@ -14,9 +14,12 @@
 
 #pragma once
 
-#include "neml2/models/Model.h"
 #include "Action.h"
 
+#ifdef NEML2_ENABLED
+#include "neml2/models/Model.h"
+#endif
+
```

Comment:
Let's see if it compiles, but I remember having to include torch (or anything that include torch) before moose headers. This has been a consequence of the moose-torch namespacing clash.

Reply:
Ahh, got it. Yeah, this was just a minor annoyance of putting the thing that is being derived from first